### PR TITLE
RAIL-3552 Custom domain validation

### PR DESCRIPTION
**Pull request by plavemar**
- block user to use http protocol while creating gooddata-react-app on bear backend

JIRA: RAIL-3552

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/270/ : SUCCESS in 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-oss-check/218/ : SUCCESS in 6s

File: src/inquiries.js [hunk 1]
```diff
@@ -35,10 +35,15 @@ export const inquireHostname = isBear =>
                 when: () => isBear,
             },
             {
-                message: "Insert your hostname",
+                message: "Insert your hostname. Protocol defaults to https if none is provided.",
```
Comment (no23reason) [inline src/inquiries.js:38]: Dose this defaulting also happen on tiger? If not please make this second sentence show only on bear.

Comment (plavemar) [reply src/inquiries.js:38]: It does. If no protocol specified, it defaults to https. It http protocol provided on tiger, it stays the same.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-merge-compliance-check/102/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/271/ : SUCCESS in 25s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-fossa-scanning-zuul-el7/67/ : SUCCESS in 37s (non-voting)

**Pull request merged**